packages/stable/src/types.ts

```diff
@@ -2380,6 +2380,7 @@ export type SingleCogniteCapability =
   | { templateInstancesAcl: AclTemplateInstances };
 
 export type SinglePatch<T> = { set: T } | { setNull: boolean };
+export type SinglePatchRequired<T> = { set: T };
```

Comment:
This is a very specific type to mt task, define it inline in the Channel patch
```
    parentId?: { set: number };
    ```

Reply:
It's not visible in the changes, but we already have SinglePatchRequiredString defined just below it 
https://github.com/cognitedata/cognite-sdk-js/blob/f408bfbf83efd9268d94e3d9b962cd64e50d35c4/packages/stable/src/types.ts#L2387

So I thought it made sense to have it there for the more generic SinglePatch as well. 
Wdyt? Should I still move it inline?